pkg/kube/kubehttp.go

```diff
@@ -54,7 +54,7 @@ var (
 	GVRApp            schema.GroupVersionResource = wego.GroupVersion.WithResource("apps")
 	GVRKustomization  schema.GroupVersionResource = kustomizev1.GroupVersion.WithResource("kustomizations")
 	GVRGitRepository  schema.GroupVersionResource = sourcev1.GroupVersion.WithResource("gitrepositories")
-	GVRHelmRepository schema.GroupVersionResource = helmv2.GroupVersion.WithResource("helmrepositories")
+	GVRHelmRepository schema.GroupVersionResource = sourcev1.GroupVersion.WithResource("helmrepositories")
```

Comment:
:thinking: Wouldn't have expected this to change

Reply:
It was broken before. Flux hasn't moved the helm repository to the new api version yet. When I was running some manual tests, I couldn't remove helm apps because they couldn't be found. I dug into the flux code a bit and discovered the bug.